server/app/src/main/java/com/ams/app/model/Absentdata.java

```diff
@@ -0,0 +1,38 @@
+package com.ams.app.model;
+import java.sql.Timestamp;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import org.hibernate.annotations.CreationTimestamp;
+import lombok.Data;
+
+@Table(name="Absentdata")
```

Comment:
table name should follow snake case
This can be absent_data

Reply:
Modified all tables to lower snake case